coverage/artifacts/artsTest/artsTest_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package artsTest
+
+import (
+	"path"
+	"testing"
+
+	"k8s.io/test-infra/coverage/io"
+)
+
+func TestLocalInputArts(t *testing.T) {
```

Comment:
It may be easier to read and reason about tests if they are entirely self-contained. To that end, use of struct literals wherever possible instead of calling out to functions like `LocalInputArtsForTest()` is preferrable

Reply:
Done